src/__tests__/Info.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import Details from '../components/Info';
+
+test('renders Details component correctly', () => {
+  const { container } = render(
+    <Provider store={store}>
+      <Details />
+    </Provider>,
+  );
+  expect(container.firstChild).toMatchSnapshot();
+});
```

Comment:
- [ ] Please kindly read through this [React Documentation](https://redux.js.org/usage/writing-tests) as regards writing tests. Integration test and unit tests for pure functions in your Redux code are still missing. ℹ️ 

<img width="585" alt="Untitled" src="https://github.com/otmaneechchafyky/react-capstone-project/assets/103386183/b83dfb4c-0b9d-4ae5-8204-5b1794ce28ac">